sdk/daml-lf/language/daml-lf.bzl

```diff
@@ -105,48 +105,37 @@ def lf_versions_aggregate(versions):
 # in a stable LF version.
 lf_docs_version = lf_version_configuration.get("preview", lf_version_configuration.get("latest"))
 
-# All LF dev versions
-LF_DEV_VERSIONS = [
-    "2.dev",
-]
-
-# All LF versions
-LF_VERSIONS = [
-    "2.1",
-] + LF_DEV_VERSIONS
-
-# All LF versions suported by the engine
-ENGINE_LF_VERSIONS = [
-    "2.1",
-    "2.dev",
-]
+# fully supported LF dev versions
```

Comment:
"Fully" here means supported by both the compiler and the engine? Or does it mean "as opposed to versions supported by the dar reader"? 

Would you mind adding a comment making the meaning explicit?

Reply:
Good advice